src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -17,9 +17,9 @@ public static class TreeExtensions
         /// Predict a target using a decision tree regression model trained with the <see cref="FastTreeRegressionTrainer"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="RegressionCatalog"/>.</param>
-        /// <param name="labelColumn">The label column.</param>
-        /// <param name="featureColumn">The feature column.</param>
-        /// <param name="weights">The optional weights column.</param>
+        /// <param name="labelColumn">The name of the label column.</param>
+        /// <param name="featureColumn">The name of the feature column.</param>
+        /// <param name="weights">The name of the optional weights column.</param>
```

Comment:
so weightsColumnName, labelColumnName, featureColumnName.

Reply:
Should we change the name of the parameters too or is a different issue?

Reply:
If it's not hard for you.
This weights is actually `weightsColumn`
or `weightColumn`

@sfilipi why we use singular for feature column and plural for weight column?

---
In reply to: [254624678](https://github.com/dotnet/machinelearning/pull/2432#discussion_r254624678) [](ancestors = 254624678)